sirepo/job_supervisor.py

```diff
@@ -757,6 +747,7 @@ def _create_op(self, opName, req, **kwargs):
             req_content=req.copy_content(),
             task=asyncio.current_task(),
         )
+#TODO(robnagler) move into _Op()
```

Comment:
Yes, this should be moved. There would be some attributeError issues if the op was destroyed before this was called.

Reply:
See https://github.com/radiasoft/sirepo/issues/2475